src/Tests/Tests/CommonOptions/DistanceUnit/DistanceUnits.doc.cs

```diff
@@ -101,5 +102,36 @@ public void UsingDifferentUnits()
 			*/
 			Expect("45.5nmi").WhenSerializing(new Distance(45.5, Nest.DistanceUnit.NauticalMiles));
 		}
+		[U]
+		public void DifferentLocales()
```

Comment:
Can use `UseCultureAttribute` on test method, and separate out into separate unit tests. Since these are in a `*.doc.cs` file, these should also be marked with `// hide` to not be emitted in generated documentation e.g.

```c#
// hide
[U]
[UseCulture("sv-SE")]
public void SerializesWithInvariantCulture()
{
   // ..implementation
}
```

Reply:
I tried to follow the pattern of the other GH issue repro cases where the issue number is part of the unit test file name.